src/docs/reference/core/Apis.GraphQL.Abstractions/README.md

```diff
@@ -176,6 +178,120 @@ Shown in the example above, we have an autoroutePart argument, this is registere
 
 Done.
 
+### Extending Content Type Query Filters
+
+In the previous section, we demonstrated how to create custom filters for more complex requirements. However, if you need to implement filtering based on your index atomic values, the `WhereInputObjectGraphType` and `IIndexAliasProvider` are what you need.
```

Comment:
I'm a bit unclear about the goals of this section. Please explain more, why you'd use this approach, perhaps with examples mentioned in a paragraph (not more code examples).